scripts/docstrings_checker_test.py

```diff
@@ -84,3 +282,26 @@ def func():
 
         exceptions = docstrings_checker.possible_exc_types(raise_node)
         self.assertEqual(exceptions, set(['Exception']))
+
+    def test_docstringify_with_okey_docstring(self) -> None:
+        valid_docstring = """Docstring that is correctly formated
+            according to the Google Python Style Guide.
+
+            Args:
+                test_value: bool. Just a test argument.
+            """
+        is_valid = isinstance(
```

Comment:
Also changed this variable name after the review

Reply:
Nice